# Request (Goal)
uint8 GREET=0
uint8 FAREWELL=1

uint8 command
uint8 repeat_count
float32 repeat_delay_sec  # Delay between repetitions in seconds
---
# Response (Result)
bool success
uint8 executed_count  # Number of successful plays before stopping
string message
---
# Feedback (stream)
# No feedback!